### PR TITLE
Modules examples

### PR DESCRIPTION
**Pull request by YustinaKvr**
1) Base operations with some infrastructure entities with short explanation and tips

### CODE REVIEW

**Comment by gtema** [comment]
recheck

File: doc/examples/3_autoscaling.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  register: as_inst_list
+  tags:
+    - get_list
+
+# Besides creating instances directly from AS group module, you can add already existing ECSs to the
+# AS group. Please pay attention that instances to be added must be in the same AZ as AS group.
+- name: Add AS instances to the AS group
+  opentelekomcloud.cloud.as_instance:
+    scaling_group: "{{ as_group_new.as_group.id }}"
+    scaling_instances: ["{{ ecs1.server.id }}", "{{ ecs2.server.id }}"]
```
Comment (anton-sidelnikov) [inline doc/examples/3_autoscaling.yaml:73]: scaling_instances: 
 - "{{ ecs1.server.id }}"
 - "{{ ecs2.server.id }}"

File: doc/examples/5_cce_cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+# Let's create a Cloud Container Engine cluster and attach it to the previously deployed
+#  infrastructure.
+- name: Create CCE Cluster
+  opentelekomcloud.cloud.cce_cluster:
```
Comment (anton-sidelnikov) [inline doc/examples/5_cce_cluster.yaml:5]: add version

Comment (anton-sidelnikov) [reply doc/examples/5_cce_cluster.yaml:5]: and more parameters

File: doc/examples/5_cce_cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    availability_zone: "eu-de-01"
+    autoscaling_enabled: False
+    cluster: "{{ cluster.cce_cluster.id }}"
+    data_volumes:
+      - volumetype: "SSD"
+        size: 120
+      - volumetype: "SATA"
+        size: 100
+        encrypted: False
+        cmk_id: ""
```
Comment (anton-sidelnikov) [inline doc/examples/5_cce_cluster.yaml:33]: remove

Comment (anton-sidelnikov) [reply doc/examples/5_cce_cluster.yaml:33]: or add additional example with cmk_id

File: doc/examples/5_cce_cluster.yaml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    initial_node_count: 0
+    k8s_tags:
+      mytag: "myvalue"
+      mysecondtag: "mysecondvalue"
+    min_node_count: 1
+    max_node_count: 3
+    network: "{{ network_id }}"
+    priority: 2
+    os: "{{ os_cluster_name }}"
+    scale_down_cooldown_time: 5
```
Comment (anton-sidelnikov) [inline doc/examples/5_cce_cluster.yaml:44]: remove

File: doc/examples/6_loadbalancer.yaml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  register: lb
+
+# This module just integrate public and private keys into certificate for listener. Both public
+#  and private keys you should obtain before on third-party resources, for instance, Letsencrypt,
+#  and put in available for Ansible engine place.
+- name: Create certificate for HTTPS connections
+  opentelekomcloud.cloud.lb_certificate:
+    name: "elb_https_cert"
+    type: "server"
+    admin_state_up: True
```
Comment (anton-sidelnikov) [inline doc/examples/6_loadbalancer.yaml:18]: remove, this is constant value

File: doc/examples/6_loadbalancer.yaml [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    content: "/home/user/files/rootCA.pem"
+    private_key: "/home/user/files/rootCA.key"
+  register: elb_cert
+
+# For every type of protocol you can create its own listener. In case of HTTP listener, please pay
+#  attention on your subnet addresses pool, it must be sufficient for all the instances including
+#  listener itself.
+- name: Create listener for HTTPS traffic
+  opentelekomcloud.cloud.lb_listener:
+    admin_state_up: True
```
Comment (anton-sidelnikov) [inline doc/examples/6_loadbalancer.yaml:28]: rm

File: doc/examples/1_initial_infra.yaml [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+# Network argument here is not an Network created on previous step, but constanta for OTC.
+#      openstack.cloud.os_router:
+#        name: "{{ router_name }}"
+#        state: present
+#        network: admin_external_net
+#        enable_snat: true
+#        interfaces:
+#          - net: "{{ network.network.name }}"
+#            subnet: "{{ subnet.subnet.name }}"
+#      register: router
```
Comment (anton-sidelnikov) [inline doc/examples/1_initial_infra.yaml:58]: uncomment pls

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: doc/examples/10_relation_database.yaml [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+# In this debug you can see all the flavors of the choosen DB type, and now you can decide what
+# flavor exactly fits your needs
+- name: debug
+  ansible.builtin.debug:
+    msg: "{{ rds_flavors.rds_flavors[0].name }}"
+
+# Now let's create RDS instance. You can locate it in two or more availability zones.
+# Password you pass to the module handles in secure mode: this means that it won't be shown in
+# module's output. Please pay attention that automatic backup strategy is setting here, in the
+# moment of creating RDS instance.
```
Comment (anton-sidelnikov) [inline doc/examples/10_relation_database.yaml:20]: add comment about cmk creation

File: doc/examples/8_monitoring.yaml [hunk 9]
```diff
@@ -0,0 +1,109 @@
+---
+
+# Now we'll create several alarms to watch our infrastructure. Mind that 'alarm_name' is given by
+# user, and 'dimensions' name and 'metric_name' are constant for each kind of resource, and can be
+# taken from API reference. SMN topic here has been created beforehand.
```
Comment (anton-sidelnikov) [inline doc/examples/8_monitoring.yaml:5]: add url to namespaces info

File: doc/examples/9_cloud_search.yaml [hunk 10]
```diff
@@ -0,0 +1,46 @@
+---
+
+# Here we'll create Cloud Search cluster contained 1 node, where system protected with Master key
+# created beforehand. Please pay attention that backup strategy is setting up also in this module.
```
Comment (anton-sidelnikov) [inline doc/examples/9_cloud_search.yaml:4]: add comment about cmk creation

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: doc/examples/1_initial_infra.yaml (hunk 7 above)
Comment (YustinaKvr) [reply doc/examples/1_initial_infra.yaml:58]: It's special section, contains examples using native OpenStack modules, to let users understand mismatches in entyties naming

**Review by anton-sidelnikov — APPROVED** [review]

**Review by vladimirvshivkov — APPROVED** [review]

**Pull request merged**